.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: node_js
 node_js:
-  - 8
```

Comment:
This should not have been removed since v2 of this package still supports Node.js 8.
https://github.com/metarhia/common/blob/a2d886800384df1a3687babb56d8dd80c913defa/package.json#L40